### DISCUSSION TITLE
Alt Text

### DISCUSSION THREAD

**Discussion by nickader** [greenelab/lab-website-template#290] [Q&A] [answered]
### Checks

- [X] I have searched **[the docs](https://greene-lab.gitbook.io/lab-website-template-docs)**, [existing issues](https://github.com/greenelab/lab-website-template/issues), and [existing discussions](https://github.com/greenelab/lab-website-template/discussions) for answers first.

### Link to your website repo

https://github.com/nickader/AderLabWebsite/tree/main

### Version of Lab Website Template you are using

1.2.1

### Description

Does anyone know how to add alt text to an image when using the figure class? Thanks!

**Comment by vincerubinetti** [comment, marked as answer]
By default the image gets alt text of whatever [caption](https://greene-lab.gitbook.io/lab-website-template-docs/basics/components/figure) you specify. If you want to customize this behavior, you can do so in `_includes/figure.html`.
